src/bin/cargo/commands/install.rs

```diff
@@ -16,6 +16,7 @@ use cargo_util::paths;
 pub fn cli() -> Command {
     subcommand("install")
         .about("Install a Rust binary")
+        .arg_dry_run("Perform all checks without installing (unstable)")
```

Comment:
I'm thinking this would go better next to `--force`.  Thoughts?

Reply:
I agree, i've moved it next to `--force`.